Dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04 as ubuntu
```

I wonder if there is a reason to use Ubuntu and not just the python one like 
```suggestion
FROM python:3.11.4-slim-bullseye
```